android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -269,8 +270,28 @@ class DVCClient private constructor(
      * [defaultValue] is set on the Variable and used to provide a default value if the Variable
      * could not be fetched or does not exist
      */
+    fun variable(key: String, defaultValue: String): Variable<String> {
```

Comment:
removing this
```kotlin
fun <T: Any> variable(key: String, defaultValue: T): Variable<T> {
        Variable.getAndValidateType(defaultValue)
        val variable = this.getCachedVariable(key, defaultValue)
```
breaks flutter sdk, 
since we need to specify which type of variable we have on variable call and saving the return value in variableUpdates breaks typing